index.js

```diff
@@ -16,8 +17,133 @@ mongoose
     return Recipe.deleteMany()
   })
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+    return Recipe.create({title: 'margharita', level: 'difficult', ingredients: ['tomato', 'mozzarella', 'ham', 'cheese'], cuisine: 'Italian', dishType: 'pizza', duration: 30, creator: 'Ruben'})
+  })
+  .then(() => {
+
+    app.get('./data', (req, res, next) => { ; //--> to retrieve data. The code included in here is a middleware.
+      res.send("prodcuts");
+    }) 
```

Comment:
That is throwing an error, the `app` in other examples is a calling from `express` that hasn't been defined or installed here and there is a `;` that would cause problems too => All the data is already imported at `const data = require('./data');`